packages/kbn-unified-field-list/src/containers/unified_field_list_sidebar/field_list_sidebar.tsx

```diff
@@ -73,6 +73,13 @@ export type UnifiedFieldListSidebarCustomizableProps = Pick<
    * Custom logic for determining which field is selected
    */
   onSelectedFieldFilter?: GroupedFieldsParams<DataViewField>['onSelectedFieldFilter'];
+
+  /**
+   * Prop to pass additional field groups to the field list
+   */
+  additionalFieldGroups?: {
+    smartFields: FieldsGroup<DataViewField>['fields'];
```

Comment:
```suggestion
    smartFields?: FieldsGroup<DataViewField>['fields'];
```

Reply:
Also we could define this type in `types` fiel and reuse here and in `use_group_fields.ts`

Reply:
That would be a bit complicated given we have generics which will then need to be passed here. Given the limited time, i will keep them like this for now